src/packages/debug/src/util.ts

```diff
@@ -0,0 +1,6 @@
+export function removeISODate(str: string): string {
+  return str.replace(
+    /\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+([+-][0-2]\d:[0-5]\d|Z)/gim,
```

Comment:
```suggestion
    /\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+([+-][0-2]\d:[0-5]\d|Z)/i,
```

If the ISO date is on the first line and at the beginning you probably want to remove global and multiline?

Reply:
Nope, it's not just in the first line, as we're running it on a multiline stringified json.

Reply:
And `i` never hurts :)

Reply:
Ok 👍